### DISCUSSION TITLE
Is it possible to create login form in modal and display error message in this modal window?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5152] [Q&A Core]
Dropdown solution for login form is not the best one.  With any error message is displayed on wrong place, it is closed after wrong log in...  When everything is ok, it is nice, but with any error.... it is not enough. 

The correct solution should be: to display the login form in modal (then you don't need that iframe for login page) and standalone login page. 

I don't ask for any core changes there are other issues.  This could be plugin area.  I know (I suppose that I know :)|  that display error message in correct place needs javascript and ajax.  

It is discussed for example here:
https://craftcms.stackexchange.com/questions/29430/login-modal-how-to-handle-incorrect-password-credentials

Any ideas in this topic? Thanks

**Comment by Alex-e107nl** [comment]
Well you could use [https://github.com/biati-digital/glightbox](https://github.com/biati-digital/glightbox) there are possibillities with events and closing the modal with a change, the only thing you need then is a link and when te underlying page is refreshing close the modal...

You can even do it with a theme... :-)

I hope that the dev's have a better solution!

**Reply by Alex-e107nl** [reply]
Oke done it with the theme.... click on the link 'Inloggen' (not the dropdown) at [https://e107.nl/](https://e107.nl/) :-)

**Reply by Jimmi08** [reply]
Wow,  I will look at it asap, but I tested it on e107.nl. Very well done.
